### ISSUE TITLE
change statement for open() is splited into two part in middle of sentence.

### ISSUE THREAD

**Issue by 5b68c441-639c-4083-a0a1-85f8c4fa2978** [python/cpython#74078]
BPO | [29892](https://bugs.python.org/issue29892)
--- | :---
Nosy | @osamu0329nakamura, @DimitrisJim, @Mariatta
PRs | <li>python/cpython#802</li><li>python/cpython#810</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = 'https://github.com/Mariatta'
closed_at = <Date 2017-03-25.03:06:14.463>
created_at = <Date 2017-03-23.23:46:18.684>
labels = ['3.7', 'docs']
title = 'change statement for open() is splited into two part in middle of sentence.'
updated_at = <Date 2017-03-25.03:06:14.462>
user = 'https://github.com/osamu0329nakamura'
```

bugs.python.org fields:
```python
activity = <Date 2017-03-25.03:06:14.462>
actor = 'Mariatta'
assignee = 'Mariatta'
closed = True
closed_date = <Date 2017-03-25.03:06:14.463>
closer = 'Mariatta'
components = ['Documentation']
creation = <Date 2017-03-23.23:46:18.684>
creator = 'OSAMU.NAKAMURA'
dependencies = []
files = []
hgrepos = []
issue_num = 29892
keywords = []
message_count = 5.0
messages = ['290070', '290071', '290459', '290460', '290461']
nosy_count = 4.0
nosy_names = ['docs@python', 'OSAMU.NAKAMURA', 'Jim Fasarakis-Hilliard', 'Mariatta']
pr_nums = ['802', '810']
priority = 'normal'
resolution = 'fixed'
stage = 'resolved'
status = 'closed'
superseder = None
type = None
url = 'https://bugs.python.org/issue29892'
versions = ['Python 3.6', 'Python 3.7']
```

</p></details>

**Comment by 5b68c441-639c-4083-a0a1-85f8c4fa2978** [comment]
In https://docs.python.org/3.6/library/functions.html#open , 
Following sentence is wrongly separated by extra asterisk.

```
FileExistsError is now raised if the file opened in exclusive
creation mode ('x') already exists.
```

This mistake is introduced by https://github.com/python/cpython/commit/3929499914d47365ae744df312e16da8955c90ac#diff-30d76a3dc0c885f86917b7d307ccf279

**Comment by b78688f9-6435-4192-9fe9-2f86f7e61fcf** [comment]
Indeed, nice catch! Submit a PR for it if you want to (if not, someone else will pick it up soon :-)

**Comment by Mariatta** [comment]
New changeset 29540cdf6c66df9f806375a95078c0c63192ef78 by Mariatta (NAKAMURA Osamu) in branch 'master':
bpo-29892: Fix wrong markup on doc-lib-functions (GH-802)
https://github.com/python/cpython/commit/29540cdf6c66df9f806375a95078c0c63192ef78

**Comment by Mariatta** [comment]
New changeset 5dafaece67cb576ef96579fb5ddcbf62e0827b0d by Mariatta in branch '3.6':
bpo-29892: Fix wrong markup on doc-lib-functions (GH-802) (GH-810)
https://github.com/python/cpython/commit/5dafaece67cb576ef96579fb5ddcbf62e0827b0d

**Comment by Mariatta** [comment]
PR was merged and backported into 3.6.

Thanks all :)
